packages/manager/modules/telecom-universe-components/src/editable-service-name/editable-service-name.html

```diff
@@ -1,57 +1,82 @@
 <form
-    class="editable-service-name"
     name="editableServiceNameForm"
-    data-ng-submit="$ctrl.saveServiceName()"
+    data-ng-submit=":: $ctrl.saveServiceName()"
 >
-    <h1>
-        <!-- NOT IN EDITION -->
+    <div
+        data-ng-hide="$ctrl.inEdition"
+        data-ng-class="{ 'd-flex': !$ctrl.inEdition }"
```

Comment:
⚠️ You are using helpers from Bootstrap 4 but it's not defined as a dependency nor a peer dependency of this module.

- `d-flex`
- `h-100`
- `px-2`
- `align-middle`